io/io/src/TFile.cxx

```diff
@@ -76,7 +76,8 @@ The structure of a directory is shown in TDirectoryFile::TDirectoryFile
 #include <errno.h>
 #include <sys/stat.h>
 #ifndef WIN32
-#   include <unistd.h>
+#include <unistd.h>
```

Comment:
Still needed?

Reply:
I removed this on CentOS and it still compiled, I will try to remove, let's see. I only re-indented because clang-format complained.

Reply:
unrelated to this PR though?

Reply:
I overlooked the `-` part of the diff. I really thought that the include had been added here so yes, it's unrelated.